node/src/actors/chain_manager/mod.rs

```diff
@@ -1736,7 +1698,7 @@ impl ChainManager {
             // Return all the blocks from this epoch range
             let hashes: Vec<(Epoch, Hash)> = block_chain_range.collect();
 
-            Ok(hashes)
+            hashes
```

Comment:
Could you remove the let hashes? Or the collect forces you to use it?

Reply:
Let's try